server/src/main/java/org/elasticsearch/action/search/TransportOpenPointInTimeAction.java

```diff
@@ -213,6 +213,11 @@ SearchPhase openPointInTimePhase(
                 searchRequest.getMaxConcurrentShardRequests(),
                 clusters
             ) {
+                @Override
+                protected String missingShardsErrorMessage(StringBuilder missingShards) {
+                    return "Search rejected due to missing shards. [open_point_in_time] action requires all shards to be available.";
```

Comment:
Maybe remove the prefix `"Search rejected due to missing shards."` and add `"missing shards[" + missingShards +"]"` at the end of the message?